app/gui2/shared/ast/tree.ts

```diff
@@ -638,7 +638,7 @@ export class OprApp extends Ast {
     return this.module.checkedGet(this.fields.get('rhs')?.node)
   }
 
-  *concreteChildren(verbatim?: boolean): IterableIterator<NodeChild> {
+  *concreteChildren(_verbatim?: boolean): IterableIterator<NodeChild> {
```

Comment:
If this is unused, can it be removed completely

Same for all following functions.

Reply:
@kazcw ?

Reply:
Some types use it, and they all have to have compatible signatures. It will be used more when we have more smart-printing.